javascript.js

```diff
@@ -0,0 +1,62 @@
+const homeButton = document.querySelector('.index__button')
+const bookmarksButton = document.querySelector('.bookmarks__button')
+const createButton = document.querySelector('.ownquestion__button')
+const profileButton = document.querySelector('.profile__button')
+
+const mainSiteHome = document.querySelector('.main__site__home')
+const mainSiteBookmarks = document.querySelector('.main__site__bookmarks')
+const mainSiteCreate = document.querySelector('.main__site__create')
+const mainSiteProfile = document.querySelector('.main__site__profile')
+
+const headerSiteHome = document.querySelector('.header__site__home')
+const headerSiteBookmarks = document.querySelector('.header__site__bookmarks')
+const headerSiteCreate = document.querySelector('.header__site__create')
+const headerSiteProfile = document.querySelector('.header__site__profile')
```

Comment:
Please reconsider the class names. Either add an `id` to the elements and get them through that or add a `data` attribute: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes
Would look like that in HTML:
```
<main class="main" data-js="home">....</main>
``` 
And like this in JS:
```js
const mainSiteHome = document.querySelector('[data-js="home"]')
```
This way you'd have your classes for CSS styling only and the selection with JavaScript in a separate attribute. Hope that makes sense.

Reply:
Yes, either `id`s  or best with the `data-js` attribute.